common/schemas/combined.js

```diff
@@ -0,0 +1,88 @@
+const {
+  activitiesDashboard,
+  activitySummary,
+  activityDescription,
+  apdOverview,
+  assurancesAndCompliance,
+  activityCostAllocationFFP,
+  activityCostAllocationOther,
+  // costAllocateFFPQuarterly, can't use because it validates against a calculated value
```

Comment:
Should we remove this? Or is this waiting on a fix?

Reply:
I'll remove it, good catch